crypto/crypto.go

```diff
@@ -233,7 +233,7 @@ func aesCBCDecrypt(key []byte, cipherText []byte, iv []byte) (plainText []byte,
 	decrypter.CryptBlocks(paddedPlainText, cipherText)
 	plainText = PKCS7Unpad(paddedPlainText)
 	if plainText == nil {
-		err = errors.New("Decryption failed: PKCS7Unpad failed after decryption")
+		err = errors.New("Decryption failed: PKCS7Unpad failed after AES decryption")
```

Comment:
Note that this error message is displayed to users. Maybe it should say "could not decrypt private key".
Nobody cares about the fact that it uses `PKCS7Unpad`.